src/routes/listMessagesService.js

```diff
@@ -1,40 +1,55 @@
 var express = require('express');
 var router = express.Router();
 var validationMessage = require('../utils/validations');
-var PersonDB = require('../model/personModel.js');
-var ListMessagesDB = require('../model/listMessagesModel.js');
+var PersonDB = require('../model/personModel');
+var ListMessagesDB = require('../model/listMessagesModel');
+var MessagesDB = require('../model/messageModel');
 
 router.get('/', function (req, res, next) {
     var datoListMessage = req.body;
     var validaciones = new validationMessage();
 
+    var listMessageWithPersonNameJson = { "listMessagesData": [] };
     ListMessagesDB.find(function (err, listMessages) {
         if (err) {
             res.status(500).send(err)
         } else {
-            res.send(listMessages);
+            PersonDB.find(function (err, person) {
+                if (err) {
+                    res.status(500).send(err)
+                } else {
+                    for (var i in listMessages) {
+                        var listMessageWithPersonName = { "_id": "", "printed": "", "year": "", "receiverId": "", "name": "" };
```

Comment:
Change the use of `var`, for `let` and `const`, and research what is the difference and why is better to use `let` and `const`

Reply:
fix